### DISCUSSION TITLE
TSX how to connect and how to use?

### DISCUSSION THREAD

**Discussion by mihail727** [nuxt/nuxt#17894] [Questions] [answered]
Please help... I would like to enter into variable pieces of html code and then render these pieces in the Vue component
For example:
`@/composables/useTitle.ts`
```ts
export const useTitle = () => {
    return {
        title: <h3 class="text-9xl">Some title</h3>
    }
}
```
`@/pages/index.vue`
```html
<template>
    <div>
        {{ useTitle().title }}
    </div>
</template>
```

**Comment by Megarsh** [comment, marked as answer]
Don't forget to wrap the title string in quotes.

If you want to use a composable for this you will need to define the title in the script setup first

```
<script setup>
    const { title } = useTitle()
</script>
```
```
<template>
    <div v-html="title"></div>
</template>
```

Or alternatively you can create a plugin and use it directly in the template https://v3.nuxtjs.org/docs/directory-structure/plugins
